### PR TITLE
Javascript filter

### PR DESCRIPTION
**Pull request by Tomash996**
js filter

### CODE REVIEW

File: src/Block.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+public class Block {
```
Comment (vasylboliuk) [inline src/Block.java]: This is not part of this pull request

File: src/Fighter.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
+public class Fighter {
```
Comment (vasylboliuk) [inline src/Fighter.java]: This class as well not a part of this pull request

File: src/JavascriptFilter.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+public class JavascriptFilter {
+        public static void main(String[] args) {
+            String[][] credentials = {{"foo", "foo@foo.com"}, {"bar_", "bar@bar.com"}};
```
Comment (vasylboliuk) [inline src/JavascriptFilter.java]: Move all solution to new method e.g searchNames 
In method main just call this method with parameters

File: src/JavascriptFilter.java [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+            for (int i = 0; i <= credentials.length - 1; i++) {
+                for (int j = 0; j <= credentials.length - 1; j++) {
+                    if (credentials[i][0].contains("_")){
```
Comment (vasylboliuk) [inline src/JavascriptFilter.java]: Does this solution pass in codewars?
As I see here should work endWith("_") is it?

File: src/Block.java (hunk 1 above)
Comment (Tomash996) [reply src/Block.java]: duplicated is deleted

File: src/Fighter.java (hunk 2 above)
Comment (Tomash996) [reply src/Fighter.java]: duplicated is deleted

File: src/JavascriptFilter.java (hunk 4 above)
Comment (Tomash996) [reply src/JavascriptFilter.java]: Couldn't check on CodeWars, as this task requires JS.
But I resolved an issue.
